app/components/enrolment_confirmation_component/enrolment_confirmation_component.html.erb

```diff
@@ -0,0 +1,28 @@
+<% if @current_user %>
+  <% if @programme.enrolment_confirmation_required? %>
+    <div class="enrolment-confirmation-component">
+      <%= render ModalComponent.new(
+        title: "Enrol on the #{@programme.certificate_name}",
+        reopen_button_text: (@button_text || "Enrol"),
```

Comment:
Could we have a method in the component called button text? As we do this || twice, be nice to have it in one place.